.aws/src/events-schema/queueCheckDelete.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Scheduled event to trigger Account Delete Monitor to start
+ * the cascade of queued events for checking whether a user's data has
+ * been fully removed after they requested to delete their account.
+ */
+ import { Resource, Fn} from 'cdktf';
+ import { Construct } from 'constructs';
+ import { eventbridgeschemas } from '@cdktf/provider-aws';
+ import { SCHEMA_REGISTRY, SCHEMA_TYPE } from './types';
+ import { SchemasSchemaConfig } from '@cdktf/provider-aws/lib/eventbridgeschemas/schemas-schema';
+ import { config } from '../event-rules/account-delete-monitor/config'
+ 
+ export class QueueCheckDeleteSchema extends Resource {
+   constructor(
+     scope: Construct,
+     private name: string,
+   ) {
+     super(scope, name);
+     this.createQueueCheckDeleteSchema()
+   }
+ 
+   private createQueueCheckDeleteSchema() {
+     const schemaProps: SchemasSchemaConfig = {
+       name: config.queueCheckDelete.schema,
+       description: `scheduled event to start "check delete" cascade, for determining if a user's data has been removed after deleting their account`,
+       type: SCHEMA_TYPE,
+       registryName: SCHEMA_REGISTRY,
+       content : Fn.jsonencode(this.getScheduleCheckDeleteSchema()),
+     };
+     const schema = new eventbridgeschemas.SchemasSchema(
+      this,
+      `${config.queueCheckDelete.scheduleExpression}-schema`,
+      schemaProps);
+     return schema;
+   }
+ 
+   /***
+    * Schema scaffold from the aws console
+    * @private
+    */
+   private getScheduleCheckDeleteSchema() {
+     return  {
+       "openapi": "3.0.0",
+       "info": {
+         "version": "1.0.0",
+         "title": "Event"
+       },
+       "paths": {},
+       "components": {
+         "schemas": {
+           "Event": {
+             "type": "object",
+             "properties": {
+              // Optional for manually generated events, 
+              // to put a boundary on what requests we check by date
+               "latestRequestDate": {
```

Comment:
this event has only `date` in the payload, 

and we will have a follow-up event to communicate from ADM->ListApi with userIds in the payload?

Reply:
We can't actually put a payload in the scheduled event as far as I can tell, but I thought it might be useful for manual triggers. We'll have to rely on the downstream logic to determine which requests to check.